Onboarding/Controller/NewVC.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NewVC.swift
+//  SubscriptionBox
+//
+//  Created by Rafa Vazquez on 2/24/21.
+//
+
+import UIKit
+
+class NewVC: UIView {
```

Comment:
Hey Rafa, I see you're trying to create a new View Controller but your NewVC is of type UIView when it should be of type ViewController.

Reply:
I just changed it that was why it was not loading when I tried adding it to my tabbar function. Thanks!